packages/api/src/index.ts

```diff
@@ -57,25 +57,48 @@ passport.use(
     {
       clientID: env.githubClientId,
       clientSecret: env.githubSecret,
+      scope: 'user:email',
       callbackURL: `${env.appUrl}/api/auth/github/callback`,
     },
     async (accessToken: any, refreshToken: any, profile: any, done: any) => {
       const currentUser = await authEm?.findOne(User, {
         githubId: profile.id,
       });
+
       if (!currentUser) {
-        logger.info('Creating new user.');
-        const newUser = new User({
-          name: profile.username,
-          githubId: profile.id,
-          hireable: false,
-          purpose: '',
-        });
-        await authEm?.persistAndFlush(newUser);
-        done(null, { profile, githubToken: accessToken });
+        try {
+          const response = await fetch('https://api.github.com/user/emails', {
+            method: 'GET',
+            headers: {
+              Authorization: `token ${accessToken}`,
+            },
+          });
+          const responseData = await response.json();
+
+          for (const emailResponse of responseData) {
+            // for loop to go through the emails and find one to match to, otherwise save null.
+            if (emailResponse.email.endsWith('.edu') && emailResponse.verified) {
+              logger.info('Creating new user.');
+              const newUser = new User({
+                name: profile.username,
+                githubId: profile.id,
+                hireable: false,
+                purpose: '',
+                email: emailResponse.email, // send the .edu email here
+              });
```

Comment:
Apparently emails are case sensitive in GitHub...
```suggestion
          for (const emailResponse of responseData) {
            // for loop to go through the emails and find one to match to, otherwise save null.
           const email = emailResponse.email.toLowerCase();
           
            if (email.endsWith('.edu') && emailResponse.verified) {
              logger.info('Creating new user.');
              const newUser = new User({
                name: profile.username,
                githubId: profile.id,
                hireable: false,
                purpose: '',
                email,
              });
```